src/components/media-loader.js

```diff
@@ -533,12 +539,6 @@ AFRAME.registerComponent("media-pager", {
     }
   },
 
-  remove() {
```

Comment:
Why did this get removed?

Reply:
`this.toolbar` is a reference to the toolbar subelement nested a few deep under `this.el` (as per the query selector), which will be removed automatically when the pdf is removed. this routine also caused a bug for some reason, where a toolbar remained floating at the origin after a pdf was removed. (my only guess is that its because we're removing a nested subelement manually, during a remove, which seems unwise since the outer loop is recursively walking the tree presumably)